docs_app/content/tutorial/4-what-is-an-operator.md

```diff
@@ -0,0 +1,71 @@
+# 4. What Is An Operator?
+
+## TLDR:
+
+- Observables are a container for a collection of values over time.
+- Operators are functions that take an [`Observable`](API) and return a new [`Observable`]. `(source: Observable<A>) => Observable<B>`.
+- Operators are applied through the [`pipe` method](API).
+- RxJS's "operators" are actually higher-order functions, that return [operator functions](GL).
+- There are A LOT of possibilities for operators.
+
+## Overview
+
+To understand what operators are, why they are useful, and why there are so many of them, it's important to understand at a very high level, that an [observable](GL) is a collection of values over time. You have this thing, your observable, and you can call [`subscribe`](API), and tell it to start pushing values to you, and it can [complete](GL) and tell you when it's done, or you can [`unsubscribe`](GL) and tell it when you no longer want values. It is, a collection that gives you a sort of linear access to its values, over time.
```

Comment:
Nice: "You have this thing, your observable, and you can call [`subscribe`](API), and tell it to start pushing values to you, and it can [complete](GL) and tell you when it's done, or you can [`unsubscribe`](GL) and tell it when you no longer want values."